posts/2018-02-09-serverless-and-graphql-in-love.md

```diff
@@ -0,0 +1,560 @@
+---
+title: Serverless and GraphQL in Love  
+description: This blog explores a beautiful relationship between Serverless and GraphQL.  
+date: 2018-02-13  
+layout: Post  
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/graphql.jpeg  
+authors:  
+  - SiddharthGupta
+---
+
+![alt text](https://user-images.githubusercontent.com/1587005/36030132-d45feefe-0d5a-11e8-986c-92dad62f3408.png "Serverless and GraphQL family")
+
+## Introduction
+
+Over the last four years, I've been exploring the world of big data, building real-time and batch systems at scale. For the last couple of months, I've been building products with serverless architectures here at Glassdoor.
+
+Given the intersection here of serverless and big data, here are few questions on everyone's mind: 
+
+1) How can we build low latency APIs to server these complex, high dimensional and big datasets which can scale on demand?
+3) Can we fire single query which can construct a nested response from multiple data sources?
+4) Is there an easy way to secure the endpoint, paginate through the data, aggregate the results at scale and what not? 
+5) Can we pay per query execution rather than running fleet of servers?
+
+The answer? GraphQL.
+
+> “Engineers like to solve problems. If there are no problems handily available, they will create their own problems.” - Scott Adams
```

Comment:
This quote/shift came suddenly--it seemed like you were gearing up to tell us how you solved your big data problems with Serverless + GraphQL, but then we segued.

Here are a few ways we could structure the intro, please tell me which makes the most sense to you:
1) "I work with serverless and big data, here's how we used GraphQL to solve our problems" (In which case the example must relate to this)
2) "As developers, we often make things more complicated; here's one way you can make your life easier"
3) "I'd like to announce the Serverless-GraphQL plugin and how you can use it"

Reply:
@andrea-p I think the quote was going with this statement which was moved to next section "I’m going to start this off with a statement of truth, to all my fellow engineers: The way we currently build APIs, where all of them are split up and maintained separately, isn’t optimal. Luckily for us, the tech horizon is ever-expanding. We have options. And we should use them."

I think it does make sense to put the above sentence in What is GraphQL? section, but we need to figure out if quote is needed or can be moved somewhere else possibly in the same section but its already too long